### PR TITLE
Add sdk and cli example for keyvault

### PR DESCRIPTION
**Pull request by xanwal**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

-Add sdk and cli example for binary payloads

fixes #

### CODE REVIEW

File: cli/deploy-moe-keyvault.sh [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+fi
+
+# Get key
+echo "Getting access key..."
+KEY=$(az ml online-endpoint get-credentials -n $ENDPOINT_NAME --query primaryKey -o tsv )
+
+# Get scoring url
+echo "Getting scoring url..."
+SCORING_URL=$(az ml online-endpoint show -n $ENDPOINT_NAME --query scoring_uri -o tsv )
+echo "Scoring url is $SCORING_URL"
```
Comment (shohei1029) [inline cli/deploy-moe-keyvault.sh:78]: Do we need to re-use these codes when we create a doc article based on the example?
If so, we may wrap the code with blanket <hoge> and </hoge>.
If not, forget my comment.

**Review by shohei1029 — COMMENTED** [review]
Thanks for the PR!

**Comment by shohei1029** [comment]
The runner seems to be failed.

**Review by dem108 — APPROVED** [review]
Great example. Thanks a lot!

**Review by shohei1029 — APPROVED** [review]

**Pull request merged**